plugins/header_rewrite/operators.cc

```diff
@@ -403,6 +403,14 @@ OperatorSetRedirect::initialize(Parser &p)
   require_resources(RSRC_RESPONSE_STATUS);
 }
 
+void
+OperatorSetRedirect::initialize_hooks()
+{
+  add_allowed_hook(TS_HTTP_READ_RESPONSE_HDR_HOOK);
+  add_allowed_hook(TS_HTTP_SEND_RESPONSE_HDR_HOOK);
+  add_allowed_hook(TS_REMAP_PSEUDO_HOOK);
```

Comment:
These are the only 3 hooks on which this operator actually works currently.  (By default, without a hook condition, operators execute on the READ_RESPONSE hook.)